host-controller/src/main/java/org/jboss/as/host/controller/DirectoryGrouping.java

```diff
@@ -52,8 +51,10 @@
     }
 
     public static DirectoryGrouping forName(String localName) {
-        final DirectoryGrouping directoryGrouping = localName != null ? MAP.get(localName.toLowerCase()) : null;
-        return directoryGrouping == null ? DirectoryGrouping.valueOf(localName.toUpperCase(Locale.ENGLISH)) : directoryGrouping;
+        if (localName == null) {
+            return null;
+        }
+        return MAP.get(localName.toLowerCase());
```

Comment:
This change is introducing an unexpected behavior change.

For example, of `localName` was something different of `null` and `MAP.get(localName.toLowerCase())` returns null, this method was returning `DirectoryGrouping.valueOf(localName.toUpperCase(Locale.ENGLISH))`, but now it will return null.

Reply:
@yersan Hi Yeray, thanks for your feedback. It's fixed. Could you review and approve, please?

Reply:
Done, @boris-unckel, thanks!